controllers/cloud.redhat.com/providers/iqe/impl.go

```diff
@@ -129,6 +129,17 @@ func createIqeContainer(j *batchv1.Job, nn types.NamespacedName, cji *crd.ClowdJ
 		TerminationMessagePolicy: core.TerminationMessageReadFile,
 	}
 
+	// attach sel-downalods volume to access Downloads from selenim container
+	j.Spec.Template.Spec.Volumes = append(j.Spec.Template.Spec.Volumes, core.Volume{
+		Name:         "sel-downloads",
+		VolumeSource: core.VolumeSource{EmptyDir: &core.EmptyDirVolumeSource{Medium: "Memory"}},
```

Comment:
Can we try to set a `SizeLimit` on this empty dir? Otherwise the memory-backed volume is apparently allowed to grow to 1/2 of the node's available RAM